src/reducers/index.js

```diff
@@ -235,14 +248,18 @@ export const widgets = (state = initialState, action) => {
         },
       };
 
-    default:
+    case SAVE_LAYOUT_CHANGE:
       return {
         ...state,
-        byId: {
```

Comment:
I don't think this is gonna work @Dave-Gallup. Since now, by 'Default', i.e. actions that you don't know about, i.e. ones that are to be handled by the widgets, are no longer sent to widget reducers.